scheduler/db/scheduler.changelog-master.yaml

```diff
@@ -3,13 +3,14 @@ databaseChangeLog:
    - changeSet:
       id: 1
       author: Fujitsu
+      dbms: postgresql
       changes:
         - createTable:
             tableName: app_scaling_specific_date_schedule
             columns:
               - column:
                   name: schedule_id
-                  type: bigint
+                  type: bigserial
```

Comment:
is there any `upgrade` test executed to validate whether the `type` changes take effect on an existing db?

Reply:
Made changes directly in the original changelog will cause error when upgrading db. So we add the changes in new changesets.

And the `bigserial` is not the real data type,  it can not use `ALTER ..`  to change the type. 

According to the document https://www.postgresql.org/docs/9.6/datatype-numeric.html#DATATYPE-SERIAL, we add the default value for the column`schedule_id` to `nextval('app_scaling_recurring_schedule_seq')`.

I did the `upgrade` test for the `type` changes:
steps:
1) Run the original liquibase changelog file and create the `autoscaler` db and also the related tables.
2) Run the modified changelog file and check the data type.
```
autoscaler=# \d app_scaling_specific_date_schedule;
                                      Table "public.app_scaling_specific_date_schedule"
           Column           |            Type             | Collation | Nullable |                  Default
----------------------------+-----------------------------+-----------+----------+-------------------------------------------
 schedule_id                | bigint                      |           | not null | nextval('schedule_id_sequence'::regclass)
```
3. Run unit test and all test case passed.